spec/integration/init_web_spec.cr

```diff
@@ -9,7 +9,7 @@ end
 describe "Initializing a new web project" do
   it "creates a full web app successfully" do
     puts "Web app: Running integration spec. This might take awhile...".colorize(:yellow)
-    should_run_successfully "crystal run src/lucky.cr -- init.custom test-project test-project"
```

Comment:
no clue why we had this doubled on all these specs lol

Reply:
![britney-spears-confused](https://user-images.githubusercontent.com/22394/76687385-cb485a00-65f9-11ea-93c2-c1c9834bfb58.gif)